src/app/jobs/weatherevents.job.ts

```diff
@@ -0,0 +1,83 @@
+// import { Mode } from '../../common/types';
+import { Job } from '../../common/job';
+import { IContainer } from '../../common/types';
+import { TextChannel } from 'discord.js';
+
+export class WeatherEventsJob extends Job {
+  // Run once an hour
+  public interval: number = 1000 * 60 * 60;
+  public name: string = 'weather_events';
+
+  constructor() {
+    super();
+  }
+
+  public async execute(container: IContainer) {
+    const channel = (container.guildService.getChannel("weather_events")) as TextChannel;
+    const resp = await container.httpService.get("https://api.weather.gov/alerts/active?area=FL");
+    const data: Weather_Event = resp.data;
+
+    if (!data.features) {
+      return;
+    }
+
+    const map: string[] = data.features.map(f => f.properties.headline);

```

Comment:
You should filter by severities, we dont need to know every time there is a little bit of thunder

Reply:
That's true. I've tried the API on quite a few states and it seems that the severity level can be one of 'Minor', 'Moderate', 'Severe', 'Extreme'. It can also be 'Unknown', but I think that's just because the weather's still developing at that point. I think Moderate is when it starts catching thunderstorms and past that things get worse. Should probably draw the line on Severe or Extreme.